### PR TITLE
deprecate mstmip_vars

### PR DESCRIPTION
**Pull request by dlebauer**
in favor of standard_vars. Documentation for the standard_vars table says that mstmip_vars are deprecated.

Currently, a call to `nc_var <- PEcAn.utils::mstmip_vars[PEcAn.utils::mstmip_vars$Variable.Name == name, ]` gives an error b/c mstmip_vars are not documented in the PEcAn.utils package as is standard_vars in base/utils/R/datasets.R

### CODE REVIEW

**Comment by jessicaguo** [comment]
Tested, works as expected.

**Comment by moki1202** [comment]
@dlebauer I am working on `utils` package atm and one of the `notes/warnings` when I run command checks on `utils` i related to `mstmip_vars`. Will it be possible to merge this PR in develop any time soon?

**Comment by dlebauer** [comment]
@moki1202 we should be able to merge if there are no other changes requested

**Comment by moki1202** [comment]
This looks pretty good to me.

**Comment by mdietze** [comment]
I'm happy to merge as soon as (A) all the Checks pass and (B) the PR is no longer tagged WIP

**Comment by infotroph** [comment]
To fix CI errors:

* convert `vd == "na"` to `is.na(vd)` on [line 57 of base/utils/R/utils.R](https://github.com/PecanProject/pecan/pull/2760/files#diff-296112e719b8455a8be63e9cac941111d4ae1700081aa05b3a9adfa8d5b9738aR54)
* Delete the string `‘mstmip_local’ ‘mstmip_vars’`, and only that string, from lines [112](https://github.com/PecanProject/pecan/blob/develop/base/utils/tests/Rcheck_reference.log#L112) and [116](https://github.com/PecanProject/pecan/blob/develop/base/utils/tests/Rcheck_reference.log#L116) of `base/utils/tests/Rcheck_reference.log`

File: modules/data.atmosphere/R/load.cfmet.R [hunk 1]
```diff
@@ -71,10 +71,10 @@ load.cfmet <- function(met.nc, lat, lon, start.date, end.date) {
   
   results <- list()
 
-  utils::data(mstmip_vars, package = "PEcAn.utils", envir = environment())
+  utils::data(standard_vars, package = "PEcAn.utils", envir = environment())
 
   ## pressure naming hack pending https://github.com/ebimodeling/model-drivers/issues/2
-  standard_names <- append(as.character(mstmip_vars$standard_name), "surface_pressure")
+  standard_names <- append(as.character(standard_vars$standard_name), "surface_pressure")
```
Comment (infotroph) [inline modules/data.atmosphere/R/load.cfmet.R:77]:
```suggestion
  standard_names <- append(as.character(PEcAn.utils::standard_vars$standard_name), "surface_pressure")
```

File: modules/data.atmosphere/R/load.cfmet.R [hunk 2]
```diff
@@ -71,10 +71,10 @@ load.cfmet <- function(met.nc, lat, lon, start.date, end.date) {
   
   results <- list()
 
-  utils::data(mstmip_vars, package = "PEcAn.utils", envir = environment())
+  utils::data(standard_vars, package = "PEcAn.utils", envir = environment())
```
Comment (infotroph) [inline modules/data.atmosphere/R/load.cfmet.R:74]:
```suggestion
 ```

**Comment by dlebauer** [comment]
Everything addressed, just waiting on a successful build! 🤞🏻

**Review by mdietze — APPROVED** [review]

**Pull request merged**